drivers/pds/lib/utils.go

```diff
@@ -805,7 +810,7 @@ func DeleteK8sDeployments(deployment string, namespace string) error {
 
 // DeleteDeployment deletes the given deployment
 func DeleteDeployment(deploymentID string) (*state.Response, error) {
-	resp, err := components.DataServiceDeployment.DeleteDeployment(deploymentID)
+	resp, err := Components.DataServiceDeployment.DeleteDeployment(deploymentID)
 	if err != nil {
 		log.Errorf("An Error Occured while deleting deployment %v", err)
 		return nil, err
```

Comment:
Don't log error and then return it, just return verbose error

Reply:
Done and for more code clean up work end to end as part of https://portworx.atlassian.net/browse/PA-678 ticket.